lib/ingredients/NgAnnotate.js

```diff
@@ -0,0 +1,35 @@
+const Ingredient = require('./Ingredient');
+const ngAnnotate = require('ng-annotate');
+const fs = require('fs');
+
+class NgAnnotate extends Ingredient {
+    constructor(config, options) {
+        super(config, options);
+
+        this.ingredientConfig = Object.assign({
+            single_quotes: true,
+            remove: true,
+            add: true
+        }, this.getIngredientConfig('NgAnnotate'));
+    }
+
+    mergeJsContent(files, templateCache = '') {
+        return files.map(function(file) {
```

Comment:
what about reading the files asynchronously to speed things up? can the `run` function return a promise?